docs/en/Distributed-Event-Bus.md

```diff
@@ -185,6 +185,28 @@ If you perform **database operations** and use the [repositories](Repositories.m
 
 > The handler class must be registered to the dependency injection (DI). The sample above uses the `ITransientDependency` to accomplish it. See the [DI document](Dependency-Injection.md) for more options.
 
+## Event Tracking and Logging
+
+It is possible to be informed when the ABP Framework **receives** or **sends** a distributed event. The framework automatically **logs** all distributed events to the standard [logger](https://docs.abp.io/en/abp/latest/Logging), but you may require additional functionality.
```

Comment:
>The framework automatically **logs** all distributed events to the standard [logger](https://docs.abp.io/en/abp/latest/Logging), but you may require additional functionality.

Are you sure about that. If so, I think we should not log it. At least, we should not log the event object, which may contain sensitive data. Also, remove the "but you may require additional functionality" part (my personal suggestion).